src/environments/array.js

```diff
@@ -450,10 +452,17 @@ defineEnvironment({
     props: {
         numArgs: 1,
     },
-    handler: function(context, args) {
-        let colalign = args[0];
-        colalign = colalign.value.map ? colalign.value : [colalign];
-        const cols = colalign.map(function(node) {
+    handler(context, args) {
+        // Since no types are specified above, the two possibilities are
+        // - The argument is wrapped in {} or [], in which case Parser's
+        //   parseGroup() returns an "ordgroup" wrapping combinations of
+        //   "mathord"s ("l", "c") or "textord"s ("|").
+        // - The argument is a bare "mathord" or "textord".
+        const ordNode = checkNodeTypes2(args[0], "mathord", "textord");
```

Comment:
This whole mathord/textord thing is a little weird.  We should probably have:
```
{
  type: "ord",
  mode: "math" | "text",
  nucleus: ...
}
```
This change can happen in a future PR though.

Reply:
That's a good idea. I agree.

Reply:
Actually, this should just be `atom` now instead of `ord`. When I rebased on latest head, some change resulted in a `rel` type object here. So I removed `assert/checkNoteTypes2` and used `assert/checkSymbolNodeType` instead.

Reply:
yes, you're right, maybe:
```
type: "atom", 
kind: "ord" | "bin" | "rel" | ..., 
mode: "math" | "text",
nucleus: ...
```